Files/Scripts/Options/The Sealed Palace.psm1

```diff
@@ -293,8 +325,13 @@ function CreateTabColors() {
 
 
     # COLORS #
+    CreateSpinAttackColorOptions
     CreateSwordTrailColorOptions -Duration
-
+    CreateReduxTextBox -Name "AlphaTip1" -Length 3  -Text "Tip Alpha Start" -Value 255 -Min 0 -Max 255 -Info "Set the starting tip transparency of the sword trail" -Credits "Anthrogi"
```

Comment:
Not a fan of this, I would rather see this and it's code changes removed

Reply:
Same reason previous, I will show it first.